core/controllers/review_tests.py

```diff
@@ -40,6 +74,38 @@ class ReviewTestsPageDataHandler(base.BaseHandler):
     """
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {
+        'classroom_url_fragment': {
+            'schema': {
+                'type': 'basestring',
+                'validators': [{
+                    'id': 'has_length_at_most',
+                    'max_value': constants.MAX_CHARS_IN_CLASSROOM_URL_FRAGMENT
+                }]
```

Comment:
Can we use regex too? I think we have some strict rules for characters in the URL fragments.

Reply:
Used regex for url fragments from constants.ts([here](https://github.com/oppia/oppia/blob/12b339a875dfc04ab31931e987b1185beee33ada/assets/constants.ts#L5681)), please take a look.